src/components/TodoItem.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { OrderItems } from '../types/Type';
+import { useTodoContext } from '../context/TodosContext';
+
+interface Props {
+  todo: OrderItems;
+}
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const { orderItems, setOrderItems } = useTodoContext();
+  const [query, setQuery] = useState(todo.title);
+  const [isEditing, setIsEditing] = useState(false);
+
+  const hendlerDestroyOne = (ItemId: number) => {
+    const visibleTodos = [...orderItems];
+
+    setOrderItems(visibleTodos.filter(values => values.id !== ItemId));
+  };
+
+  const handleClickKeybord = (event: React.KeyboardEvent, ItemID: number) => {
```

Comment:
You inside single todo, you don't need to pass `id` to this function as you already have it